DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/device_global/README.md

```diff
@@ -92,6 +93,7 @@ int main () {
   q.copy(val, &x).wait(); // Read from device_global into x
 }
 ```
+>**Note**: `sycl::queue::copy()` only works in IP Authoring flow since Intel does not ship a BSP that supports a dedicated interface for accessing a `device global`.
```

Comment:
```suggestion
>**Note**: `sycl::queue::copy()` currently only works in the IP Authoring flow since Intel does not ship a BSP that supports a dedicated interface for accessing a `device global`.
```